src/libostree/ostree-repo.h

```diff
@@ -1081,7 +1081,7 @@ gboolean ostree_repo_prune_from_reachable (OstreeRepo             *self,
 /**
  * OstreeRepoPullFlags:
  * @OSTREE_REPO_PULL_FLAGS_NONE: No special options for pull
- * @OSTREE_REPO_PULL_FLAGS_MIRROR: Write out refs suitable for mirrors
+ * @OSTREE_REPO_PULL_FLAGS_MIRROR: Write out refs suitable for mirrors and fetch all refs if none requested
```

Comment:
I think we need even more detail as noted in https://github.com/ostreedev/ostree/issues/1100. Is this the actual text that shows up in the man page or is this just a code comment?